docs/api/api.yml

```
@@ -0,0 +1,83 @@
+openapi: 3.0.0
+info:
+  title: Download Service API
+  version: 1.1.2
+  description: |
+    This document specifies the Download Service API. It provides a direct interface to download files from the archive.
+    This API implements the operations `GET` on the paths `/` and `/ark:/:NAAN/:uuid/*`.
+    The NAAN represents the configured 'name assigning authority number' which for DANS is 73189.
+    It is not neccessary for this service to supply the bag store the bag is stored in.
+
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0
+
+servers:
+  - url: 'http://localhost:20160'
+  - url: 'https://datavault.dans.knaw.nl:20160'
+
+paths:
+  /:
+    get:
+      summary: Returns a description of the service.
+      description:
+        This is the starting place for navigating the Download Service. It returns a description of the service.
+      responses:
+        200:
+          description: Description of the service
+          content:
+            text/plain:
+              schema:
+                type: string
+                description: A plain text, human readable description of the service
+                example: EASY Download Service running v1.1.2.
+
+  /ark:/{NAAN}/{FileId}:
+    get:
+      summary: Returns an output stream of the file specified by the file-id.
+      description: |
+        The file which is fetched is specified by the file-id.
+        The file-id is defined as the bag-id followed by the percent encoded relative path of the file in the bag.
+        This service requires the item-id to refer to a payload file, it is unable to download directories and non-payload files:
+
+        ```
+        example valid file-ids:
+         40594b6d-8378-4260-b96b-13b57beadf7c/path/to/file
+         40594b6d-8378-4260-b96b-13b57beadf7c/path/to/another%20file
+        ```
+
+      parameters:
+        - $ref: '#/components/parameters/FileId'
+        - $ref: '#/components/parameters/NAAN'
+      responses:
+        200:
+          description: The file stream will be returned.
```

```suggestion
          description: OK. Successfully returning the requested file.
```

It seems irrelevant to me that the file might be returned as a stream.